.babelrc

```diff
@@ -8,11 +8,11 @@
 		}],
 		"stage-2"
 	],
-	"plugins": ["transform-runtime"],
+	"plugins": ["transform-vue-jsx", "transform-runtime"],
```

Comment:
Was this needed for a testing framework?

Reply:
Updates from the vue-cli webpack template that were missed when the transform-vue-jsx was added to the dependencies.